.github/workflows/wheel.yaml

```
@@ -18,14 +30,25 @@ jobs:
     env:
       CIBW_TEST_COMMAND: python -c "import numcodecs"
       CIBW_BUILD: "cp310-* cp311-* cp312-* cp313-*"
-      CIBW_SKIP: "pp* *-musllinux_* *win32 *_i686 *_s390x"
+      CIBW_ARCHS_MACOS: native
+      CIBW_ARCHS_WINDOWS: native
```

Do we want to try the same thing for Windows ARM?

```suggestion
      CIBW_ARCHS_WINDOWS: "AMD64 ARM64"
```

Basing this off [this doc]( https://cibuildwheel.pypa.io/en/stable/options/#archs )